Northwind/Northwind/Config/LogWriter.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Reflection;
+
+namespace Northwind.Config
+{
+    public class LogWriter
+    {
+        private string m_exePath = string.Empty;
+        public LogWriter(string logMessage)
```

Comment:
I think it's a good practice to leave one empty line between constructor and field declarations.